.github/actions/base-setup/action.yml

```diff
@@ -25,19 +25,19 @@ runs:
         environment-file: ${{ inputs.server_folder }}/environment.yml
 
     - name: Install Node.js
-      shell: bash
+      shell: bash -l {0}
       run: |
         mamba install nodejs=14 yarn
 
     - name: Install the server
-      shell: bash
+      shell: bash -l {0}
       run: |
-        pip install -e .
+        python -m pip install -e .
       working-directory: ${{ inputs.server_folder }}
 
     - name: Install the frontend
       shell: bash
```

Comment:
`bash -l {0}` may be missing here as well?
We can also define it as part of the defaults settings: https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#example-set-the-default-shell-and-working-directory

Reply:
Thanks for the tip. Apparently it is not supported within a custom action definition